grype/db/schema.go

```diff
@@ -1,5 +1,3 @@
 package db
 
-import v5 "github.com/anchore/grype/grype/db/v5"
-
-var SchemaVersion = v5.SchemaVersion
+const SchemaVersion = 6
```

Comment:
Should this just be entirely removed? We should be reading the version from the DB (we are, at least mostly)

Reply:
To avoid confusion and multiple constants in v6.ModelVersion and db.SchemaVersion, i've removed this